ProcessMaker/processmaker
# feature/FOUR-10083: Submit Actions in Pane Navigation

## Issue & Reproduction Steps
As a Participant, I want to have the ability to perform the actions that are available when a Task is open in the preview pane. (the same thata will happen when I open teh task in the normal way). This ensures that the completion of the Task will be done inside the pane.

For example if there is a Form that has to be submited, this “Submit Button” should work and give the mesagge of correct or wrong ejecution.

## Solution
- https://processmaker.atlassian.net/browse/FOUR-10063

## How to Test
1. Create a request in the TASK
2. Open the preview
3. Check the screen related and make changes
4. Click on the Submit

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-10083

## Code Review Checklist

## Environment Fall
ci:next
